src/node.rs

```diff
@@ -5,6 +5,11 @@ use serde::Serialize;
 
 use crate::js_doc::JsDoc;
 
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub struct NamespaceDef {
+  pub elements: Vec<DocNode>,
+}
```

Comment:
Didn't seem worth it to have a separate file for just this... actually we may want to consider collapsing many of these modules into here and parser.rs.